bin/locale_update.py

```diff
@@ -35,7 +41,15 @@ def create_locale(src_dir):
         # Convert loop-client-l10n repo names to loop repo names.
         dst_dir = src_dir.replace('_', '-').replace('templates', 'en-US')
 
-        shutil.copytree(os.path.join(l10n_src, src_dir), os.path.join(l10n_dst, dst_dir))
+        # Don't copy the en-US files. Stick with what's in our tree as that might
+        # be a later version.
+        if dst_dir == "en-US":
+            return dst_dir
+
+        # Copy the l10n files, but ignore any `.keep`
+        shutil.copytree(os.path.join(l10n_src, src_dir),
+                        os.path.join(l10n_dst, dst_dir),
+                        ignore=shutil.ignore_patterns(".keep"))
```

Comment:
You could always do `git diff --cached` to check the diff as one would need to do with the other automated `git add`s.